easy/3.js

```diff
@@ -1,11 +1,30 @@
 function minMax(arr) {
-	
+    let howMany = arr.length;
+    let winner = arr[0]; 
+    let loser = arr[howMany-1];
+    
+
+    for(i = 0; i<=howMany; i++){
+        if(winner<arr[i]){
+            winner = arr[i];
+        }
+        if(loser>arr[i]){
+            loser = arr[i];
+        }
+    }
+    
+
+    let returnValueArr = [loser, winner];
+    //console.log(returnValueArr);
+    
+    return returnValueArr;

```

Comment:
You don't have to declare this variable if it exists only for store value and return it